RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotESProducer.cc

```diff
@@ -38,12 +39,14 @@ class OnlineBeamSpotESProducer : public edm::ESProducer {
   BeamSpotObjects fakeBS_;
   const int timeThreshold_;
   const double sigmaZThreshold_;
+  const double sigmaXYThreshold_;
 };
 
 OnlineBeamSpotESProducer::OnlineBeamSpotESProducer(const edm::ParameterSet& p)
     // get parameters
     : timeThreshold_(p.getParameter<int>("timeThreshold")),
-      sigmaZThreshold_(p.getParameter<double>("sigmaZThreshold")) {
+      sigmaZThreshold_(p.getParameter<double>("sigmaZThreshold")),
+      sigmaXYThreshold_(p.getParameter<double>("sigmaXYThreshold")) {
```

Comment:
```suggestion
      sigmaXYThreshold_(p.getParameter<double>("sigmaXYThreshold") * 1E-4) {
```
following the suggestion in https://github.com/cms-sw/cmssw/pull/40744/files#r1106074334

Reply:
@dzuolo can you implement this suggestion (removing the multiplications in the `isGoodBS` as pointed out above https://github.com/cms-sw/cmssw/pull/40744#discussion_r1106074334) ?

Reply:
Hi @mmusich! This comment has been addressed in https://github.com/cms-sw/cmssw/pull/40744/commits/d34fe0e6031a4a6996d4ff5587c135d5c7c6fbcd